board/stm32fx/interrupt_handlers.h

```diff
@@ -2,6 +2,7 @@
 // Only implemented the STM32F413 interrupts for now, the STM32F203 specific ones do not fall into the scope of SIL2
 
 void WWDG_IRQHandler(void) {handle_interrupt(WWDG_IRQn);}
+// cppcheck-suppress misra-c2012-5.5
```

Comment:
why did these have to be added now?

Reply:
That's temporarily, haven't looked into it yet, but get this after I added h7 resources. Seems that h7 lib has a macro for backward compatibility named the same.